### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by jeremiahsherrill** [thedevdojo/chatter#290]
Tested trying in laravel 8 and it won't work.   It's too bad, this had a lot of promise.

**Comment by skino2019** [comment]
Im after the Same Answer. Installing inside of laravel 8. get the constraint error.

**Comment by jeremiahsherrill** [comment]
@skino2019  I decided I am going to have to rewrite parts of it in order to make it work.   This project is dead, I would recommend if you are interested in it, fork and rewrite the parts that are necessary.

**Comment by skino2019** [comment]
Unfortunately i'm not smart enough for that yet :( 

thanks for reply.

**Comment by Jaroslav001** [comment]
> @skino2019 I decided I am going to have to rewrite parts of it in order to make it work. This project is dead, I would recommend if you are interested in it, fork and rewrite the parts that are necessary.

Did you manage it to work with Laravel 8?

**Comment by jeremiahsherrill** [comment]
yes, I had to basically copy the code to my project and tie everything up different, fix routes, and update some libraries it used, but I got it to work on 8

**Comment by andreaselia** [comment]
Hey, I'd say this project is pretty dead - I'm developing an update to my forum package [Laravel Firefly](https://github.com/AndreasElia/laravel-firefly), if you'd like to follow along.

**Comment by skino2019** [comment]
> Hey, I'd say this project is pretty dead - I'm developing an update to my forum package [Laravel Firefly](https://github.com/AndreasElia/laravel-firefly), if you'd like to follow along.

Cheers for reply, i hadn't seen the previous comment. i shall 100% follow the other project

**Comment by DryerLintPurple** [comment]
Maybe try one of the active forks of Chatter:

https://github.com/thedevdojo/chatter/network

(Hit Shift+➡ to scroll to the right end of the graph.)
